Diagnostics/HealthChecker/Analyzer/Invoke-AnalyzerFrequentConfigurationIssues.ps1

```diff
@@ -349,6 +349,26 @@ function Invoke-AnalyzerFrequentConfigurationIssues {
         Add-AnalyzedResultInformation @params
     }
 
+    if ($null -ne $exchangeInformation.TransportService) {
+        if ($exchangeInformation.TransportService.MaxPerDomainOutboundConnections -lt 40) {
```

Comment:
Is this something we only want to display to customers and end users when there is a confirmation issue? If yes, then this is correct.

Reply:
that was the intention, to only surface if not set to recommended values.